plugins/modules/vmware_vm_storage_policy.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Ansible Project
+# Copyright: (c) 2020, Dustin Scott <sdustin@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
```

Comment:
With latest changes `ANSIBLE_METADATA` is no longer required, so can be removed.

Reply:
Fixed in commit https://github.com/ansible-collections/vmware/pull/237/commits/cbc799151d840db04b9837d9c02f18200fc953e8